### ISSUE TITLE
Unable to create ZnapZend replication tasks within the UI on Rocky Linux

### ISSUE THREAD

**Issue by bailey-97** [45Drives/cockpit-zfs-manager#2]
**Bug Info**

When selecting the configure replication task in the ZFS tab on Rocky Linux, it returns with an error of 

"Replication task could not be configured

${filesystem}"

After which leaves the ZFS page greyed and unable to select anything further

**Steps to reproduce the behavior**

Go to the ZFS tab, and select the "Configure Replication Task" Option

**Expected behavior**

Ability to create ZFS snapshot and replication tasks within the UI

**Screenshots**
![issue](https://user-images.githubusercontent.com/88982700/143035013-063bca30-14b0-4c91-a4c4-f49e0cb52f04.gif)

**Console Output**

Failed to load resource: the server responded with a status of 404 (Not Found)
zfs.js:973 () => { } Cockpit, Package, SOS Report, Delete: Skipped
cockpit.js:1 grep: /etc/dnf/automatic.conf: No such file or directory

p @ cockpit.js:1
zfs.js:4182 () => { } Storage Pools, Refresh: In Progress, Pool: tank
zfs.js:4298 () => { } Storage Pools, Refresh: Success, Pool: tank
zfs.js:4865 () => { } File Systems, Get: In Progress, Pool: tank
zfs.js:8857 () => { } Snapshots, Get: In Progress, Pool: tank
zfs.js:9971 () => { } Status, Get: In Progress, Pool: tank
zfs.js:8870 () => { } Snapshots, Get: Success, Pool: tank
zfs.js:4519 () => { } Storage Pools, Size, Get: In Progress, Pool: tank
zfs.js:4890 () => { } File Systems, Get: Success, Pool: tank
zfs.js:11395 () => { } Disks, Lsblk, Get: In Progress
zfs.js:4524 () => { } Storage Pools, Size, Get: Success, Pool: tank
zfs.js:11400 () => { } Disks, Lsblk, Get: Success
zfs.js:9992 () => { } Status, Get: Success, Pool: tank
updates.js:2 Could not determine kpatch availability: {"detail":"higher version \"0.9.2-5.el8\" of package kpatch.noarch is already installed","code":9}
zfs.js:21186 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'length')
    at FnModalReplicationTaskCreateContent (zfs.js:21186)
RedHatDisplay-Bold.woff:1 Failed to load resource: the server responded with a status of 404 (Not Found)
DevTools failed to load source map: Could not load content for https://192.168.69.125:9090/cockpit/$d890ef116b803cb1ccd2af727e1957abcc0342a9fcd66af022b80f651284f3ce/zfs/jquery.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
2DevTools failed to load source map: Could not load content for https://192.168.69.125:9090/cockpit/$d890ef116b803cb1ccd2af727e1957abcc0342a9fcd66af022b80f651284f3ce/zfs/patternfly.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Desktop (please complete the following information)**

OS: Windows
Browser: Chrome
Version: Version 95.0.4638.69 (Official Build) (64-bit)

**Smartphone (please complete the following information)**

Device: N/A
OS: N/A
Browser: N/A
Version: N/A

**Server Side**

OS: Rocky Linux
Cockpit Version: 250

**Additional context**

N/A

**Comment by Zack-45D** [comment]
Just curious if any updates on this issue?

--Thanks all!

**Comment by bkelly16** [comment]
@bailey-97 @Zack-45D No updates. Looking into issue now.

**Comment by bkelly16** [comment]
Issue is fixed with version v0.20.0 or higher of znapzend. Rocky repos have been are updated to have latest znapzend v0.21.0.
```
dnf update znapzend
or
dnf install https://repo.45drives.com/rhel/el8/stable/x86_64/znapzend-0.21.0-1.el8.x86_64.rpm
```
